sleap_io/io/nwb.py

```diff
@@ -225,16 +198,13 @@ def instance_to_skeleton_instance(instance: Instance) -> SkeletonInstance:  # ty
     Returns:
         An NWB SkeletonInstance.
     """
-    if not isinstance(instance, Instance):
-        raise ValueError("The input must be a SLEAP Instance object.")
-    
     skeleton = slp_skeleton_to_nwb(instance.skeleton)
     points_list = list(instance.points.values())
     node_locs = [[point.x, point.y] for point in points_list]
     np_node_locations = np.array(node_locs)
     return SkeletonInstance(
-        name=name_generator("skeleton_instance"),
-        id=np.uint(10),
+        name=f"skeleton_instance_{id(instance)}",
+        id=id(instance),
```

Comment:
This `id` method will return the memory address pointer for the `instance` object. This is ok, but the numbers will be pretty crazy and random. A nicer ID system would be to use the index of the instance from a global list of instances.